packages/terra-base/package.json

```diff
@@ -28,8 +28,7 @@
     "react-dom": "^15.4.2"
   },
   "dependencies": {
-    "classnames": "^2.2.5",
-    "react": "^15.4.2"
```

Comment:
This is also the same approach cloudflare-ui is taking: https://github.com/cloudflare/cf-ui/blob/master/packages/cf-component-callout/package.json#L14